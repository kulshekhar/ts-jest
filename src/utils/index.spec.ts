test('should show warning when importing from `ts-jest/utils`', async () => {
  const consoleSpy = (console.warn = jest.fn())

  const { createJestPreset, pathsToModuleNameMapper } = await import('../../utils')

  createJestPreset()
  pathsToModuleNameMapper({})

  expect(consoleSpy).toHaveBeenCalledTimes(2)
  expect(consoleSpy.mock.calls.reduce((acc, val) => acc.concat(val), [])).toMatchSnapshot()
})
