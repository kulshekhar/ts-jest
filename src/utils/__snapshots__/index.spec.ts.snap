// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show warning when importing from \`ts-jest/utils\` 1`] = `
Array [
  "ts-jest[main] (WARN) Replace any occurrences of \\"ts-jest/utils\\" with just \\"ts-jest\\".",
  "ts-jest[main] (WARN) Replace any occurrences of \\"ts-jest/utils\\" with just \\"ts-jest\\".",
]
`;
